packages/components/src/components/Timerange/Timerange.stories.tsx

```diff
@@ -10,30 +9,34 @@ const Center = styled.div`
     padding: 100px 0px;
 `;
 
-Center.displayName = 'CenterWrapper';
+export default {
+    title: 'Misc/Timerange',
+    component: TimerangeComponent,
+};
 
-storiesOf('Misc/Timerange', module).add(
-    'Timerange',
-    () => {
-        const startDate = new Date(date('Start date', new Date('March 7 2021')));
-        const endDate = new Date(date('End date', new Date('March 13 2021')));
-        const confirm = text('Confirm button label', 'Confirm');
-        const cancel = text('Cancel button label', 'Cancel');
-
-        return (
-            <Center>
-                <Timerange
-                    startDate={startDate}
-                    endDate={endDate}
-                    onSubmit={() => {}}
-                    onCancel={() => {}}
-                    ctaSubmit={confirm}
-                    ctaCancel={cancel}
-                />
-            </Center>
-        );
+export const Timerange: StoryObj<TimerangeProps> = {
+    render: args => (
+        <Center>
+            <TimerangeComponent
+                startDate={args.startDate}
+                endDate={args.endDate}
+                onSubmit={() => {}}
+                onCancel={() => {}}
+                ctaSubmit={args.ctaSubmit}
+                ctaCancel={args.ctaCancel}
+            />
+        </Center>
+    ),
+    args: {
+        ctaSubmit: 'Confirm',
+        ctaCancel: 'Cancel',
     },
-    {
-        argTypes: { onChange: { action: 'onChange' } },
+    argTypes: {
```

Comment:
nit: add string argType for `ctaSubmit` and `ctaCance`

Reply:
b46eecb78af7218a63cc6f2855edbf94b56b3967